java/src/Ice/src/main/java/com/zeroc/Ice/PropertyNames.java

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
Maybe I should of regenerated this file with the script, but it seemed easier to just directly edit it.
Even if the file advises against it : vP

Reply:
You should fix the script.

Reply:
Yes, I did also fix the script.
Was just saying that I fixed the file by hand.